pkg/http-metrics/pkg/db/options.go

```diff
@@ -0,0 +1,58 @@
+package db
+
```

Comment:
Same thing holds for this file

Reply:
the file is not present in the `kosmos` repository. In order to not import it from another repository, I've opted to copy the needed code.

Reply:
Ok but add a TODO anyway for a future refactor